### ISSUE TITLE
Request: Landscape orientation / full scalability of the UI

### ISSUE THREAD

**Issue by chris-blues** [moritz-weber/mucke#137]
It would be nice, if the app would support landscape orientation. It is running on my RaspberryPi running LineageOS which only has a landscape orientated screen attached to it. The app looks pretty silly, using only a 1/4 of the screen width.

![20230827_115146](https://github.com/moritz-weber/mucke/assets/15331023/f3b310d4-c71a-431e-9d91-4dbd236d2bf9)

**Comment by moritz-weber** [comment]
Yeah, this looks ridiculous 😅 Allowing landscape mode actually is the default behavior. It doesn't look good though, and getting it to a satisfying point will be a lot of work. In my opinion even Spotify doesn't do this well.

I'd love to work on this, but there are more important issues regarding the functionality of the app.

Since I'm not a UI/UX designer, any help here is welcome.

**Comment by chris-blues** [comment]
Yeah, no hurry! This can wait!

I have a lot on my plate these days, but maybe I could take a look at the styling side. I'm not a UI designer or sth. But I did a few websites with CSS and stuff and maybe I can find some tweaks for this.
How is it done anyway? Also CSS?

**Comment by moritz-weber** [comment]
mucke is developed with Flutter: https://flutter.dev/
It uses no CSS at all. The whole thing is based on a widget tree that I actually find quite intuitive to work with. To get an idea, you can check out the presentation folder: https://github.com/moritz-weber/mucke/tree/master/src/lib/presentation

As a side note, I think that the styling side is not very complicated. We mainly use a simple set of colors and mostly standard components. I think the difficulty is in designing the layouts for the different form factors. Smartphone landscape is one thing, but a bigger screen might require a completely different layout to make sensible use of the available space.
